src/firebase.js

```diff
@@ -1,4 +1,13 @@
 import firebase from 'firebase'
+import 'firebase/firestore';
 import { environment } from "./environment";
-firebase.initializeApp(environment.firebase);
-export default firebase;
\ No newline at end of file
+
+const app = firebase.initializeApp(environment.firebase);
+const auth = app.auth();
+const firestore = app.firestore();
+
+export {
+    firebase,
+    auth,
+    firestore
+};
```

Comment:
There's no new line at the end of the file. I guess we will set it up in the config for the projects...

Reply:
Done